.github/workflows/renovate.yaml

```diff
@@ -0,0 +1,35 @@
+name: Renovate
+
+on:
```

Comment:
maybe add a repository dispatch as an option? That way we can trigger it from other repos if we want

Reply:
Done.